examples/InputObject.hs

```diff
@@ -48,7 +48,7 @@ example = interpretAnonymousQuery @Query root
 
 main :: IO ()
 main = do
-  response <- example "{ description(dogStuff: {toy: \"bone\", likesTreats: true}) }"
+  response <- example "{ description(dogStuff: {_toy: \"bone\", _likesTreats: true}) }"
```

Comment:
Thanks for the PR! 

I'm not entirely sure what's going on here. Is there a bug for reference? Can you maybe add a description to the commit message for why the example didn't work before and why it works with this fix?

Reply:
What breaks because of this?